packages/task/src/browser/task-configurations.ts

```diff
@@ -56,7 +58,11 @@ export class TaskConfigurations implements Disposable {
 
     constructor(
         @inject(FileSystemWatcher) protected readonly watcherServer: FileSystemWatcher,
-        @inject(FileSystem) protected readonly fileSystem: FileSystem
+        @inject(FileSystem) protected readonly fileSystem: FileSystem,
+        @inject(WorkspaceService) protected readonly workspaceService: WorkspaceService,
```

Comment:
please add new dependenices always vis property injection, otherwise it is a breaking change, see inversify conventions : https://github.com/theia-ide/theia/wiki/Coding-Guidelines#property-injection

Reply:
Fixed, thank you!